docs/package-requirements.md

```diff
@@ -0,0 +1,22 @@
+# Package Requirements
+
+A set of criteria used to evaluate packages used by [GitHub](https://github.com).
+
+* MUST be published on [npm registery](https://www.npmjs.com).
+* SHOULD be also available on [GitHub](https://github.com) with tagged releases.
+* MUST provide an [OSI](https://opensource.org/licenses) license file and declare in `package.json` `"license"` field.
+* SHOULD be available under an MIT license.
+* MUST follow [SemVer](http://semver.org) versioning.
```

Comment:
This is difficult to ascertain. A project may say that they are following SemVer, but we can't know how closely they are sticking to the principle. Or, they may be following SemVer, but not have that explicitly stated anywhere. (This is the case with my own projects.) I guess SemVer states that you should have a "semver" tag in your project, but I rarely see that in practice.